### DISCUSSION TITLE
Database for slider shapes

### DISCUSSION THREAD

**Discussion by papertek** [ppy/osu#16584] [Ideas] [answered]
I think a public database for custom slider shapes made by the community would be cool

this is for the editor since it really is a struggle to make beautiful sliders on stable and on lazer

**Comment by frenzibyte** [comment]
~~Within scope of https://github.com/ppy/osu/issues/8773.~~

Not fully part of https://github.com/ppy/osu/issues/8773 given that this asks for publicity support as well.

**Reply by papertek** [reply]
osu

**Comment by peppy** [comment, marked as answer]
Just opening a map with a slider and copy it into your map sounds fine. We'd want to support this workflow anyways.

**Reply by papertek** [reply]
so copying and pasting sliders between maps will be possible in the future? because currently, the way to do it is by making a separate difficulty that actually includes the slider or by using notepad. I'm just thinking of a way to make cool sliders less tedious to actually create

**Reply by peppy** [reply]
yep, should be.

**Reply by papertek** [reply]
👍
